GliaWidgets/Glia.swift

```diff
@@ -59,9 +126,9 @@ public class Glia {
     ///   - sceneProvider: Used to provide `UIWindowScene` to the framework. Defaults to the first active foreground scene.
     ///
     /// - throws:
-    ///   - `ConfigurationError.invalidSite`
-    ///   - `ConfigurationError.invalidEnvironment`
-    ///   - `ConfigurationError.invalidAppToken`
+    ///   - `SalemoveSDK.ConfigurationError.invalidSite`
+    ///   - `SalemoveSDK.ConfigurationError.invalidEnvironment`
+    ///   - `SalemoveSDK.ConfigurationError.invalidAppToken`
```

Comment:
I think it can also throw SalemoveSDK.invalidSiteApiKey & perhaps even SalemoveSDK.invalidRegionEndpoint? (same comment for other start method)

Reply:
you are right - fixed in 0a19888